cloudinit/net/netbsd.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit import log as logging
+from cloudinit import util
+import cloudinit.net.bsd
+
+LOG = logging.getLogger(__name__)
+
+
+class Renderer(cloudinit.net.bsd.BSDRenderer):
+
+    def __init__(self, config=None):
+        super(Renderer, self).__init__()
+
+    def write_config(self):
+        if self.dhcp_interfaces():
+            self.set_rc_config_value('dhcpcd', 'YES')
+            self.set_rc_config_value(
+                    'dhcpcd_flags',
+                    ' '.join(self.dhcp_interfaces()))
+        for device_name, v in self.interface_configurations.items():
+            if isinstance(v, dict):
+                self.set_rc_config_value(
+                    'ifconfig_' + device_name,
+                    v.get('address') + ' netmask ' + v.get('netmask'))
+
+    def start_services(self, run=False):
```

Comment:
where / how is the `run` parameter used?

Reply:
I'm not a big fan of this attribute, I think it's superfluous. The class attribute should be enough. Anyway. I just reproduce what has been done with FreeBSD. I've adjusted the code to directly use the `run` parameter. Like it's done with FreeBSD.